.circleci/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Force-push the built HTML to the `gh-pages` branch.
+#
+
+set -e
+
+DEPLOY_DIR=~/project
+
+# trust GitHub server keys
+if [ ! -d ~/.ssh/ ]; then
```

Comment:
Since we're using Bash specifically, we should default always to use `if [[ <test> ]]; then`, using `[[` instead of `[`
More info: http://mywiki.wooledge.org/BashFAQ/031

Reply:
Today I learned. Thanks for the link! This was fixed in latest commit.